keepalived/check/ipvswrapper.c

```diff
@@ -639,8 +639,14 @@ ipvs_group_sync_entry(virtual_server_t *vs, virtual_server_group_entry_t *vsge)
 				/* Talk to the IPVS channel */
 				ipvs_talk(IP_VS_SO_SET_ADDDEST, &srule, &drule, NULL, false);
 			}
-			else
+			else {
+				if (vsge->addr.ss_family == AF_INET6)
+					inet_sockaddrip6(&vsge->addr, &srule.nf_addr.in6);
+				else 
+					srule.nf_addr.ip = inet_sockaddrip4(&vsge->addr);
+
```

Comment:
The above additional 4 lines seem unnecessary since ipvs_group_range_cmd() sets the address field in the srule.

Reply:
Yes, when we go into the ipvs_group_range_cmd, there is no problem. While  if we are single one ,that is to say, vsge->range is 0, then we will miss the set of srule.nf_addr.ip. But in fact, I think maybe we just delete the judgement of  vsge->range, whether it is zero or not , it will work well : )